samples/bluetooth/mesh/mesh_dfu/target/README.rst

```diff
@@ -69,10 +69,9 @@ The models are used for the following purposes:
 * Config Server allows configurator devices to configure the node remotely.
 * Health Server provides ``attention`` callbacks that are used during provisioning to call your attention to the device.
   These callbacks trigger blinking of the LEDs.
-* The Binary Large Object (BLOB) Transfer models, :ref:`zephyr:bluetooth_mesh_blob_srv` and :ref:`zephyr:bluetooth_mesh_blob_cli`, provide functionality for sending large binary objects from a single source to many Target nodes over the Bluetooth mesh network.
-  It is the underlying transport method for the Device Firmware Update (DFU).
-  BLOB Transfer Server is instantiated on the primary element.
-* The :ref:`zephyr:bluetooth_mesh_dfu_srv` model and its base models are instantiated on the secondary element.
+* Use of Binary Large Object (BLOB) Transfer models is the underlying transport method for the Device Firmware Update (DFU).
```

Comment:
DFU acronym is already defined in the page title, so it's okay to either use the full term or the acronym, doesn't need to be both.